features/src/main/feature/camel-features.xml

```diff
@@ -285,11 +285,18 @@
 
     <feature name='camel-activemq' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-jms</feature>
+        <bundle dependency='true'>mvn:org.apache.activemq/activemq-client-jakarta/${activemq-version}</bundle>
+        <bundle dependency='true'>mvn:org.fusesource.hawtbuf/hawtbuf/1.11</bundle>
```

Comment:
This one should be optional in the import. I would remove it and define optional in the wrapper.

Reply:
With the proposed change I'm getting the following error
```
15:29:07.147 DEBUG [ActiveMQ Transport: tcp://localhost/127.0.0.1:61616@50075] Async exception with no exception listener: java.io.IOException: Unexpected error occurred: java.lang.NoClassDefFoundError: org/fusesource/hawtbuf/UTF8Buffer
java.io.IOException: Unexpected error occurred: java.lang.NoClassDefFoundError: org/fusesource/hawtbuf/UTF8Buffer
	at org.apache.activemq.transport.tcp.TcpTransport.run(TcpTransport.java:222) [!/:5.18.3]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoClassDefFoundError: org/fusesource/hawtbuf/UTF8Buffer
	at org.apache.activemq.util.MarshallingSupport.readUTF(MarshallingSupport.java:229) ~[!/:5.18.3]
	at org.apache.activemq.util.MarshallingSupport.unmarshalPrimitive(MarshallingSupport.java:200) ~[!/:5.18.3]
	at org.apache.activemq.util.MarshallingSupport.unmarshalPrimitiveMap(MarshallingSupport.java:98) ~[!/:5.18.3]
	at org.apache.activemq.util.MarshallingSupport.unmarshalPrimitiveMap(MarshallingSupport.java:78) ~[!/:5.18.3]
	at org.apache.activemq.command.WireFormatInfo.unmarsallProperties(WireFormatInfo.java:177) ~[!/:5.18.3]
	at org.apache.activemq.command.WireFormatInfo.getProperties(WireFormatInfo.java:150) ~[!/:5.18.3]
	at org.apache.activemq.command.WireFormatInfo.toString(WireFormatInfo.java:374) ~[!/:5.18.3]
	at java.lang.StringConcatHelper.stringOf(StringConcatHelper.java:453) ~[?:?]
	at org.apache.activemq.transport.InactivityMonitor.configuredOk(InactivityMonitor.java:103) ~[!/:5.18.3]
	at org.apache.activemq.transport.AbstractInactivityMonitor.startMonitorThreads(AbstractInactivityMonitor.java:449) ~[!/:5.18.3]
	at org.apache.activemq.transport.InactivityMonitor.startMonitorThreads(InactivityMonitor.java:88) ~[!/:5.18.3]
	at org.apache.activemq.transport.InactivityMonitor.processInboundWireFormatInfo(InactivityMonitor.java:61) ~[!/:5.18.3]
	at org.apache.activemq.transport.AbstractInactivityMonitor.onCommand(AbstractInactivityMonitor.java:295) [!/:5.18.3]
	at org.apache.activemq.transport.TransportSupport.doConsume(TransportSupport.java:83) [!/:5.18.3]
	at org.apache.activemq.transport.tcp.TcpTransport.doRun(TcpTransport.java:233) ~[!/:5.18.3]
	at org.apache.activemq.transport.tcp.TcpTransport.run(TcpTransport.java:215) ~[!/:5.18.3]
	... 1 more

```
I suppose we still need hawtbuf

Reply:
Oh, you are right, it's used by the marshaller.